sdk/documenttranslator/ai-document-translator-rest/samples-dev/translateFromBlob.ts

```diff
@@ -90,7 +90,7 @@ export async function main() {
 
     // The checkStatus operation returns a retry-after header that contains the
     // time in seconds to wait before sending the next polling request
-    const parsedRetryAfter = Number.parseInt(operationState.headers["retry-after"] || "5");
+    const parsedRetryAfter = Number.parseInt(String(operationState.headers["retry-after"]) || "5");
```

Comment:
```suggestion
    const parsedRetryAfter = operationState.headers["retry-after"] ?? 5;
```

Reply:
need confirm whether the retry-after should be a number or a string @MaryGao